test_go.py

```diff
@@ -23,13 +23,13 @@ def test_go_version(container):
     "container_git_clone",
     [
         GitRepositoryBuild(
-            repository_url="https://github.com/weaveworks/kured.git",
```

Comment:
Why are you changing the golang tests?

Reply:
hmmm.... i guess my repo is out of sync somehow :(